test/tizen_capi/unittest_tizen_capi.cpp

```diff
@@ -272,6 +272,44 @@ TEST(nntrainer_capi_nnmodel, addLayer_04_p) {
   EXPECT_EQ(status, ML_ERROR_NONE);
 }
 
+/**
+ * @brief Neural Network Model add layer test
+ */
+TEST(nntrainer_capi_nnmodel, addLayer_05_n) {
+  int status = ML_ERROR_NONE;
+
+  ml_nnmodel_h model = NULL;
+  ml_nnlayer_h layer = NULL;
+
+  std::string config_file = "./test_compile_01_p.ini";
+  RESET_CONFIG(config_file.c_str());
+  replaceString("Layers = inputlayer outputlayer",
+                "Layers = inputlayer outputlayer", config_file, config_str);
+
+  status = ml_nnmodel_construct_with_conf(config_file.c_str(), &model);
+  EXPECT_EQ(status, ML_ERROR_NONE);
+
+  status = ml_nnmodel_compile_with_conf(model);
+  EXPECT_EQ(status, ML_ERROR_NONE);
+
+  status = ml_nnlayer_create(&layer, ML_LAYER_TYPE_FC);
+  EXPECT_EQ(status, ML_ERROR_NONE);
+
+  status = ml_nnlayer_set_property(layer, "unit= 10", "activation=softmax",
+                                   "bias_zero=true", "weight_decay=l2norm",
+                                   "weight_decay_lambda=0.005", NULL);
+  EXPECT_EQ(status, ML_ERROR_NONE);
+
```

Comment:
I haven't thought about it. adding the layer with compiled model. Right. I think it should not allowed.